modules/concepts/controllers/admin-controllers/_index.md

```diff
@@ -182,6 +286,48 @@ In order to generate the valid URI of a controller you created from inside the m
 ```
 
 
+## Key Changes in PrestaShop 9.0 (Symfony 6.4)
+
+### Controllers as Services
+
+In Symfony 6.4, controllers must be defined as services. The container passed to controllers is no longer the "global container" but a dedicated, optimized container based on the services injected into it.
+
+**Important implications:**
+
+- The `$this->get('service_name')` method is no longer available in modern controllers
+- `FrameworkBundleAdminController` maintains backward compatibility but is **deprecated** and will be removed in PrestaShop 10.0
+- New controllers should extend `PrestaShopAdminController` which follows Symfony best practices
+
+### Dependency Injection Methods
+
+You have three main ways to inject services:
+
+1. **Constructor injection** - For services used across multiple actions
+2. **Method injection** - For services used in a single action (more optimized)
+3. **Autowire attribute** - Use `#[Autowire(service: 'service_id')]` to inject services by their ID
+
+### PrestaShopAdminController Helper Methods
+
+The new base controller provides convenient helper methods for commonly used services:
+
+- `$this->getConfiguration()` - Access configuration service
+- `$this->getTranslator()` - Access translator service
+- `$this->getRouter()` - Access router service
+- `$this->getFlashBag()` - Access flash messages
```

Comment:
Isn't this a doublon with the paragraph above that lists even more thelper methods?